.setup/pip/system_requirements.txt

```
@@ -0,0 +1,37 @@
+# Extract from install_system.sh, latest update May 7th
+
+python-pam==1.8.4
+PyYAML==3.12
+psycopg2-binary==2.8.6
+sqlalchemy<1.4.0
```

```suggestion
sqlalchemy==1.3.24
```

apply this in the other files.